ai/Algorithm/path_partitionner.py

```diff
@@ -115,8 +116,9 @@ def fastpathplanner(self, path, depth=0, avoid_dir=None):
         return path
 
     def get_path(self, player: OurPlayer, pose_target: Pose=Pose(), cruise_speed: [int, float]=1,
-                 old_path=None, old_raw_path=Path(Position(99999, 99999), Position(99999, -99999))):
+                 old_path=None, old_raw_path=Path(Position(99999, 99999), Position(99999, -99999)), end_speed=0):
```

Comment:
We should be removing arguments to this method, not adding some. Every arguments to this method is useless as every one except player are found in player. Instead of adding arguments, you can add this line after line 122:
self.end_speed = self.player.ai_command.end_speed